### PR TITLE
Update NBengineElephant.cfg

### PR DESCRIPTION
**Pull request by AtomikkuSan**
Did some performance tweaking.

* updates #41 - Adjust [NBengineElephant.cfg]

### CODE REVIEW

**Comment by AtomikkuSan** [comment]
@zer0Kerbal, I've done it. This pull request now goes to the 0.2.99.1-prerelease branch instead of the master branch.

File: GameData/NecroBones/ElephantEngine/Parts/NBengineElephant.cfg [hunk 1]
```diff
@@ -2,7 +2,7 @@ PART
 {
 	name = NBengineElephant
 	module = Part
-	author = NecroBones and zer0Kerbal
+	author = NecroBones, zer0Kerbal, Atomikku
```
Comment (zer0Kerbal) [inline GameData/NecroBones/ElephantEngine/Parts/NBengineElephant.cfg:5]: kindly put zer0Kerbal last 

1. original authors
2. subsequent authors
3. current authors in alphabetical order

**Review by zer0Kerbal — CHANGES_REQUESTED** [review]
one change requested.

File: GameData/NecroBones/ElephantEngine/Parts/NBengineElephant.cfg (hunk 1 above)
Comment (zer0Kerbal) [reply GameData/NecroBones/ElephantEngine/Parts/NBengineElephant.cfg:5]: will approve and merge once updated.

well done.

Comment (AtomikkuSan) [reply GameData/NecroBones/ElephantEngine/Parts/NBengineElephant.cfg:5]: Hmm, I always put you before me 'cause it's my project, not yours.

Comment (AtomikkuSan) [reply GameData/NecroBones/ElephantEngine/Parts/NBengineElephant.cfg]: Already done!

**Comment by AtomikkuSan** [comment]
@zer0Kerbal, can you take a look at the latest commit?

**Review by zer0Kerbal — APPROVED** [review]

**Pull request merged**